src/contexts/AuthUserContext/index.tsx

```diff
@@ -30,17 +30,17 @@ export function AuthUserProvider({
     // or when the user's ID token changes
     FirebaseService.getAuth().onIdTokenChanged((firebaseUser: FirebaseUser | null) => {
       if (firebaseUser) {
+        axios.interceptors.request.use(
+          async (request) => {
+            request.headers.authorization = await firebaseUser.getIdToken();
+            return request;
+          },
+          (err: Error) => {
+            return Promise.reject(err);
+          }
+        );
```

Comment:
The axios interceptor can go first before `UserService.getUser`. This is because say we configured to remain logged in.

I visit the site the next day, my idToken is no longer valid. The first time I go into the site, it'll call GET /users/:id, but then the axios interceptor is broken because there is no idToken attached to the request headers.

Moving interceptor before GEt /users/:id solves this problem

Reply:
Makes sense! Good catch